<?xml version="1.0"?>
<launch>
  <include file="$(find replanners_bench_cells)/launch/cell.launch">
    <arg name="demo_package" value="replanners_bench_3d_simple_moveit_config"/>
    <arg name="static_scene" value="static_scene_replanners_bench_3d_simple.yaml"/>
    <arg name="rviz"         value="$(find replanners_bench_cells)/config/rviz_crash_test_3d_simple_config.rviz"/>
  </include>
  
  <rosparam command="load" file="$(find graph_ros1)/config/test_graph_core.yaml"/>

  <node pkg="graph_ros1"
        name="test_graph_core"
        type="test_graph_core"
        output="screen"
        launch-prefix="gdb -ex run --args" >
  </node>

</launch>
